contrib/build-package.sh

```diff
@@ -175,7 +176,7 @@ case "$1" in
     REPODIR=cachelib/external/$NAME
     SRCDIR=$REPODIR/build/cmake
     external_git_clone=yes
-    external_git_branch=release
+    external_git_commit=8420502e
```

Comment:
Can we simply go with `external_git_tag`? 

This is temporary fix, so I don't think we need to add any complexity. The checkout commands for tag/commit should be identical.

Reply:
Ah, nice, that's a better solution. Will do, thanks!